src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -313,6 +326,45 @@ public void writeBody(PrintWriter w) {
         }
 
         /**
+         * Check if there is a ParameterValue with the same name
+         *
+         */
+        private boolean parameterValueExist (List<ParameterValue> parameterValues, String name) {
+            for(ParameterValue parameterValue : parameterValues) {
+                if (parameterValue.getName().equals(name)) {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        /**
+         * Get the default parameters values from a job
+         *
+         */
+        private List<ParameterValue> getDefaultParametersValues(Job job) {
+            ParametersDefinitionProperty paramDefProp = (ParametersDefinitionProperty) job.getProperty(ParametersDefinitionProperty.class);
+            ArrayList<ParameterValue> defValues = new ArrayList<ParameterValue>();
```

Comment:
Good practice is to set the initial size of the array if you can estimate it max border (like in this case)